app/models/notification.rb

```diff
@@ -3,7 +3,7 @@ class Notification < ActiveRecord::Base
   has_many :users, through: :user_notifications
   belongs_to :generator_user, class_name: 'User'
 
-  enum type_of: [:assignment, :recent_changes, :system_message]
+  enum type_of: Constants::NOTIFICATIONS_TYPES
```

Comment:
Could we move this, and all other extendable arrays etc., into a separate file (e.g. `Extends::`, or anything better if you can come up with)?